modules/order/tests/src/Kernel/Entity/OrderItemTest.php

```diff
@@ -87,7 +89,6 @@ public function testOrderItem() {
     $order_item->setUnitPrice($unit_price);
     $this->assertEquals($unit_price, $order_item->getUnitPrice());
 
-    $order_item->setQuantity('1');
```

Comment:
I am removing this setQuantity() call here in order to test Unit / Total prices below (otherwise they would be the same value, and the quantity check is already happening in the test assertions above).